### PR TITLE
Fix SSH in prod

### PR DESCRIPTION
**Pull request by mjudeikis**
### Which issue this PR addresses:

Fixes production ssh. Most of this are just small cosmetics making it to real easier. Change is how we mount host path. 

### What this PR does / why we need it:

This should fix SSH in production

### Test plan for issue:

Manually tested on shared cluster with custom image deployed and geneva action

### Is there any documentation that needs to be updated for this PR?

No

### CODE REVIEW

File: pkg/operator/controllers/routefix/routefix.go [hunk 1]
```diff
@@ -55,18 +54,18 @@ iptables -N ICMP_ACTION || true
 iptables -F ICMP_ACTION
 iptables -A ICMP_ACTION -j LOG
 iptables -A ICMP_ACTION -j DROP
-oc observe nodes -a '{ .status.addresses[1].address }' -- ` + configmapScriptDir + `/` + configmapScriptName
+/host/usr/bin/oc observe nodes -a '{ .status.addresses[1].address }' -- ` + configmapScriptDir + `/` + configmapScriptName
```
Comment (mjudeikis) [inline pkg/operator/controllers/routefix/routefix.go:57]: This is change 1

File: pkg/operator/controllers/routefix/routefix.go [hunk 2]
```diff
@@ -178,8 +176,8 @@ func (r *RouteFixReconciler) resources(ctx context.Context, cluster *arov1alpha1
 								},
 								VolumeMounts: []corev1.VolumeMount{
 									{
-										Name:      "host-slash",
-										MountPath: "/",
+										Name:      "host",
+										MountPath: "/host",
```
Comment (mjudeikis) [inline pkg/operator/controllers/routefix/routefix.go:180]: This is change 2

**Comment by mjudeikis** [comment]
I quite jet dont understand how it was working properly before. and how it works now. Due to fact I can;t find who is doing chrooting here... But this works. 
```
++ date '+%m%d %H:%M:%S.%N'
I0504 11:41:03.050373692 - drop-icmp - start drop-icmp aro-v4-shared-gxqb4-worker-eastus2-rknnp
+ echo 'I0504 11:41:03.050373692 - drop-icmp - start drop-icmp aro-v4-shared-gxqb4-worker-eastus2-rknnp'
+ iptables -X CHECK_ICMP_SOURCE
chroot: cannot change root directory to '/host': No such file or directory
+ true
+ iptables -N CHECK_ICMP_SOURCE
chroot: cannot change root directory to '/host': No such file or directory
+ true
+ iptables -F CHECK_ICMP_SOURCE
chroot: cannot change root directory to '/host': No such file or directory
```

**Comment by mjudeikis** [comment]
This works because SDN image has this:
```
sh-4.4# which iptables
/usr/sbin/iptables
sh-4.4# cat /usr/sbin/iptables
#!/bin/sh

exec chroot /host /usr/sbin/iptables "$@"
```
image

**Review by jewzaam — APPROVED** [review]

**Pull request merged**